site/realm/auth/providers.json

```diff
@@ -14,8 +14,8 @@
         "type": "local-userpass",
         "config": {
             "autoConfirm": true,
-            "resetPasswordSubject": "Password Resets are not Currently Supported",
-            "resetPasswordUrl": "http://aiid.partnershiponai.org/",
+            "resetPasswordSubject": "",
```

Comment:
@cesarvarela this should be something like "Your AI Incident Database Password Reset Request"?

Reply:
I see that this was merge, but just in case, the message you get when this is set to empty:
![image](https://user-images.githubusercontent.com/1172479/142932400-763f2d98-6c05-4b69-9fb7-d5ca58a6b85e.png)

Reply:
Thanks! Looks good.